tools/link-lib/start-lib-sync.js

```diff
@@ -0,0 +1,37 @@
+const { join } = require('path');
+const chokidar = require('chokidar');
+const copyfiles = require('copyfiles');
+const shelljs = require('shelljs');
+const debounce = require('lodash/debounce');
+
+const options = require('./start-lib.config.example.json');
+
+options.libs.forEach(lib => {
+  const out = join(options.appPath, 'node_modules/@cognizone', lib);
+  shelljs.rm('-rf', out);
+
+  const paths = [];
+  const distPath = join('./dist/libs', lib);
+
+  const copyToApp = path => {
+    paths.push(path);
+    actualCopy(paths);
+  };
+
+  const actualCopy = debounce(paths => {
+    copyfiles(
+      [...paths, out],
+      {
+        up: distPath.split('\\').length,
```

Comment:
Should use [path.sep](https://nodejs.org/docs/latest/api/path.html#pathsep) so that it will work on any OS 😀 (can you log `distPath` to be safe? I was expecting `/` even on Windows but I might be wrong 🤔 )